src/applications/personalization/profile-2/components/AccountSecurity.jsx

```diff
@@ -21,7 +21,7 @@ class AccountSecurity extends Component {
       <>
         <h2
           tabIndex="-1"
-          className="vads-u-margin-y--2 medium-screen:vads-u-margin-y--4"
+          className="vads-u-line-height--1  vads-u-margin-y--2 medium-screen:vads-u-margin-y--4"
```

Comment:
This is get the top of the section header to more closely align with the rule line at the top of the sidenav menu. Previously, with the line height of this header at its default value, the section header sat a lot lower than the sidenav menu rule line.

After:

<img width="669" alt="Screen Shot 2020-05-04 at 10 36 24 AM" src="https://user-images.githubusercontent.com/20728956/80995580-51f80700-8df3-11ea-83eb-4f7f130b5cba.png">

Reply:
previously:

![image](https://user-images.githubusercontent.com/20728956/80996134-2de8f580-8df4-11ea-8a3c-c402391908fb.png)